.github/workflows/go-interop.yml

```diff
@@ -0,0 +1,30 @@
+on: pull_request
+name: Interop
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    name: GO/JS
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.x
+      - name: Go information
+        run: |
+          go version
+          go env
+      - name: Building JS Test
+        working-directory: interop/js
+        run: npm install
+      - name: Building Go Test
+        working-directory: interop/go
+        run: go build -o test-go .
+      - name: Test
+        working-directory: interop
+        run: |
+          npm --prefix ./js/ start &
+          sleep 1
+          ./go/test-go &
+          wait -n
+          wait -n
```

Comment:
Why do we need this twice? iiuc `wait -n` waits for all background jobs to complete?

Reply:
`wait -n` waits for the next background job to complete and returns its exit code. Unfortunately, a simple `wait` won't catch the exit code.

Reply:
I'll add a comment.